packages/react-components/react-motion/library/src/hooks/useAnimateAtoms.ts

```diff
@@ -2,6 +2,9 @@ import * as React from 'react';
 import type { AnimationHandle, AtomMotion } from '../types';
 
 function useAnimateAtomsInSupportedEnvironment() {
+  // eslint-disable-next-line @nx/workspace-no-restricted-globals
+  const SUPPORTS_PERSIST = typeof window !== 'undefined' && typeof window.Animation?.prototype.persist === 'function';
```

Comment:
Not in love with having this here. Ideally it would live in global scope because `SUPPORTS_PERSIST` will be constant for any given environment.

I moved it here because I could not find a good way to test it with Jest.

Reply:
You could move this to a function check in another module and mock that moduile in the tests, but I don't mind keeping it like this `ELEMENT_SUPPORTS_WEB_ANIMATIONS` is done similarly